### ISSUE TITLE
Some skyCultures star names include non-native names.

### ISSUE THREAD

**Issue by 3domennotry** [Stellarium/stellarium#3876]
star_names.fab has 2 pattern.

One of them is the star name by native name.
One of them is the explain of star name by English.

However, These skycultures "star_names.fab" include non-native names.

hawaiian_starlines: Mirzim, Bellatrix, Alnitak, Alnilam, Mintaka
khoi-san: all star names
zulu: all star names
romanian: many, almost
egyptian: Procyon, Sirius

I think that all skycultures are ancient except for modern_XXXX.

Therefore, I think that non-native name were not used in these ancient skycultures (zulu, khoi-san, hawaiian_starlines).

Perhaps, Many non-native name (At least Acrux, Peacock) were not used in ancient romanian skycultures.

https://archive.hokulea.com/ike/hookele/hawaiian_star_lines.html
This website, native name is yellow, non-native name is blue.

skycultures/egyptian/star_names.fab include Procyon and Sirius.
Maybe, the names of these stars were used in ancient Egypt.

### System
* Stellarium version: stellarium-24.2-qt5-win32.exe
  + stellarium-24.2-98544be-patch-qt5-win32.exe
  + stellarium-24.2-623f996-patch-qt5-win32.exe
  + stellarium-24.2-19f4d98-patch-qt5-win32.exe

* Operating system: Win7 32bit

(Sorry, I can not speak English very well.)

thank you.

**Comment by gzotti** [comment]
Yes. We are in preparation of a big change in our skycultures, so please let us keep this as-is for now. 

What you see now has grown over more than twenty years. Contributions of mixed quality, people doing their best, providin better than nothing, but sometimes inaccuracies appear. Until 2017 or so we had no clear rules what to expect in the content. After that (with the User Guide) we tried to receive better quality contributions. Some SC authors have not provided proper star names, in this case the default star list was used, and it is clear in context that names collected in Europe are not original Zulu. (Maybe you then see an older version!) In other cases, some modern star names are likely used for reference (so that users still can orient themselves!) If we had a job description like "institute for cultural astronomy" with 3-5 full-time positions, we could work faster. In any case, we depend on high-quality external contributions here.

**Comment by 3domennotry** [comment]
I look forward to it.

This is my personal opinion.

I think that skycultures/anutan/star_names.fab is best.
This file contains both the native name and the description.
When I select &alpha; CMa, it shows: "Te Tino A Manu (The Bird's Body)".
This file not contains modern star names.
No problem.
Because, We know that &alpha; CMa is Sirius.

And,
I think that skycultures/(each folders)/description.en.utf8 should not contain modern star names.
Of course I know Sirius, Procyon, Betelgeuse and several major star names.
But, I do not know almost minor star names.

I reported following:
https://github.com/Stellarium/stellarium/issues/3845
https://github.com/Stellarium/stellarium/issues/3854

Caph, Shedar and Navi, The names of these stars are not well known in Japan. (I am Japanese.)

If I find a misspelling of a star name that I know of, I can report it or correct it.
But, I find a misspelling of a star name that I do not know of, I can not report it or correct it.

Please look at: skycultures/hawaiian_starlines/description.en.utf8 .

Descriptions include the names of stars such as Capella, Castor, Pollux, Procyon and Sirius.
No problem.
I know these star names.

Descriptions include the names of stars such as "&alpha; Peg, Markab", "&beta; Peg, Scheat", and "&gamma; Peg, Algenib".
No problem.
I do not know these star names.
But I can identify it by the Bayer letters.

Descriptions include the names of stars such as "Mintaka" and "Mirzim".
I'm in trouble.
I can not identify.
Fortunately, I can see the external link:
https://archive.hokulea.com/ike/hookele/hawaiian_star_lines.html
According to this website,
Mintaka is &delta; Ori.
Perhaps, Not Mirzim But Murzim.
Perhaps, Murzim is &beta; CMa.

thank you.

**Comment by 3domennotry** [comment]
I have confirmed in "25.1.148 / 25.1-5d13699 (2025-05-28) / stellarium-25.1-5d13699-qt5-win64.exe".
not fixed.

**Comment by gzotti** [comment]
Please change the files as you see fit and send a pull request.

**Comment by 3domennotry** [comment]
I can only modify these files.
I cannot replace.
thank you.

[index(hawaiian_starlines).json](https://github.com/user-attachments/files/20505543/index.hawaiian_starlines.json)
[index(khoi-san).json](https://github.com/user-attachments/files/20505544/index.khoi-san.json)
[index(romanian).json](https://github.com/user-attachments/files/20505545/index.romanian.json)
[index(zulu).json](https://github.com/user-attachments/files/20505546/index.zulu.json)
